enumeration_test.go

```diff
@@ -283,8 +283,7 @@ func TestEnumerationQueryCondition(t *testing.T) {
 		_, err = rQuery.SetDataBuffer("cols", colsBuffer)
 		require.NoError(t, err)
 		err = rQuery.Submit()
-		require.Error(t, err)
-		require.Contains(t, err.Error(), "Enumeration value not found")
+		require.NoError(t, err)
```

Comment:
This seems like a meaningful change. What’s the reason for it?

Reply:
Quoting the commit message in eeb4540:
> Core PR https://github.com/TileDB-Inc/TileDB/pull/4707 relaxed the `QueryCondition` requirement for enumeration values to be valid.  This means that where we previously expected an error, we now get no error.

Reply:
Thanks; I was reading the review message and not the original commits.